localboot/main.go

```diff
@@ -96,10 +74,9 @@ func main() {
 
 	// try to kexec into every boot config kernel until one succeeds
 	for _, cfg := range bootconfigs {
-		log.Printf("trying to boot %s", cfg.Kernel.Name())
-		if *doDryRun {
-			// note: in dry-run, this loop brea at the first entry
-			// unconditionally
+		debug("Trying boot configuration %+v", cfg)
+		if dryrun {
```

Comment:
I find it confusing to go into the loop, and then break out immediately. I would put this if outside and never even go in in dryrun mode.

Reply:
done